src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The "+ getName() + " began its work.");
```

Comment:
The message in the 'doWork' method should be more informative and indicate the specific type of Machine that is working. For example, "Bulldozer started its work."